README.md

```diff
@@ -1,5 +1,18 @@
 # Get started
 
+## Difference with [multicluster-controlplane](https://github.com/open-cluster-management-io/multicluster-controlplane) in open-cluster-management
+
+This reposotiy is an extension to [open-cluster-management-io/multicluster-controlplane](https://github.com/open-cluster-management-io/multicluster-controlplane) with the following enhancements:
+1. Added managed cluster info addon
+2. Added policy addon
+3. Added managed service account addon
+4. The above addon components are covered in Integration/E2E test cases
```

Comment:
maybe remove 4. Here we highlight features in this repo. the test cases is for guarantee the quality. what do you think?

Reply:
If this is about highlight features. It's reasonable to remove 4. 

I have removed it from here.